transformation_test/testdata/ops_agent_test-TestLogEntrySpecialFields/output_otel.yaml

```diff
@@ -1,74 +1,65 @@
 - collector_errors:
-  - caller: internal/base_exporter.go:117
+  - caller: internal/base_exporter.go:116
     error: "could not process attribute gcp.source_location: json: cannot unmarshal string into Go struct field LogEntrySourceLocation.line of type int64"
     level: error
     msg: Exporting failed. Rejecting data. Try enabling sending_queue to survive temporary failures.
     otelcol.component.id: googlecloud/otel
     otelcol.component.kind: exporter
     otelcol.signal: logs
     rejected_items: 1.0
-    resource:
-      service.instance.id: test-service-instance-id
-      service.name: otelopscol
-      service.version: ""
+    resource: {}
     stacktrace: |-
       go.opentelemetry.io/collector/exporter/exporterhelper/internal.(*BaseExporter).Send
-        go.opentelemetry.io/collector/exporter@v0.130.0/exporterhelper/internal/base_exporter.go:117
+        go.opentelemetry.io/collector/exporter@v0.126.0/exporterhelper/internal/base_exporter.go:116
```

Comment:
The updates in this file point out that it was run with an `otelopscol` build that was using `Opentelemetry` version `v0.126.0`. The update in the Ops Agent to  use`v0.130.0` is very recent (https://github.com/GoogleCloudPlatform/ops-agent/pull/1975) so the build may still be using the `v0.126.0` version.

One command that could help is `make rebuild_submodules` to make sure `otelopscol` is built with the latest version.